cmds/bmp/tasks_command_test.go

```diff
@@ -72,10 +74,57 @@ var _ = Describe("BMP: tasks command", func() {
 	})
 
 	Describe("#Execute", func() {
-		It("executes a good TasksCommand", func() {
-			rc, err := cmd.Execute(args)
-			Expect(rc).To(Equal(0))
-			Expect(err).ToNot(HaveOccurred())
+		Context("executes a good TasksCommand", func() {
+			BeforeEach(func() {
+				fakeBmpClient.TasksResponse.Status = 200
+				fakeBmpClient.TasksErr = nil
+				options = cmds.Options{
+					Verbose: false,
+					Latest:  1,
+				}
+			})
+
+			It("executes a good TasksCommand without specifying latest", func() {
+				rc, err := cmd.Execute([]string{"bmp", "tasks"})
+				Expect(rc).To(Equal(0))
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("executes a good TasksCommand with specifying latest", func() {
+				cmd = bmp.NewTasksCommand(options, fakeBmpClient)
+
+				rc, err := cmd.Execute([]string{"bmp", "tasks", "--latest=1"})
+				Expect(rc).To(Equal(0))
```

Comment:
We need to verify that one task or at a minimum something was written in the output. Maybe you can make sure output contains the correct total?